### DISCUSSION TITLE
why remove filer.options.buckets_folder ?

### DISCUSSION THREAD

**Discussion by zouyixiong** [seaweedfs/seaweedfs#6008] [Q&A] [answered]
https://github.com/seaweedfs/seaweedfs/commit/1c7e404abec01da8bb322ef4000d6acecf1a6be5

I am deploying a SeaweedFS instance with Filer S3 and Elastic Filer Store. In my setup, I will write data to a bucket daily, and this bucket will be deleted after a few days. I am handling the TTL (Time To Live) outside of the SeaweedFS system, so I will use the deleteBucket S3 API when the TTL is reached.

After some testing, I noticed that all S3 buckets are stored under the /buckets directory in the filer store, which corresponds to a single Elasticsearch index called seaweedfs_buckets. The problem is that deleting the metadata in the filer store when deleting a bucket is not efficient. I believe a better approach would be to have one Elasticsearch index per bucket, so we can delete the Elasticsearch index directly.

I then found the configuration option filer.options.buckets_folder, which seems to be what I need. However, I noticed that this parameter has been removed from the configuration (although it still works). I wonder why this parameter was removed?

**Comment by zouyixiong** [comment]
![image](https://github.com/user-attachments/assets/589b2cff-67c4-4fbf-ad8f-1ae4ccbbc849)
After config filer.options.buckets_folder = "", I got actually what I want.

**Reply by chrislusf** [reply, marked as answer]
It was a left over and would not always work.
